### ISSUE TITLE
new O

### ISSUE THREAD

**Issue by emmaccode** [ChifiSource/Olive.jl#131]
#### standard output
One aspect of this project which is surprisingly difficult is **standard output.** Julia provides a handy `redirect_stdio` method for this, but there are a number of shortcomings. This is what is currently used to get the standard output from a `Cell` in `Olive`.
- The only way to write this output into memory is to use a `Pipe`.
- redirecting the output in this way duplicates stream descriptors -- throwing an error if more than one is opened at a time.
#### proposed solution
The solution to this, hopefully, will rely on the mutation of the new `olive_module` bare modules. These modules will remove `println(a::Any) = println(stdout, a::Any ...)` and replace it with `println(a::Any ...) = println(ourbuffer, a ...)`

Basically, we are going to change the `Methods` inside of `Base` to facilitate this rather than redirecting our output.

**Comment by emmaccode** [comment]
broken pipes have been stopped, I think. Will reopen in the future if this issue arises with high concurrency.
